pkgs/by-name/go/gosmee/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+}:
+
+buildGoModule rec {
+  pname = "gosmee";
+  version = "0.21.0";
+
+  src = fetchFromGitHub {
+    owner = "chmouel";
+    repo = "gosmee";
+    rev = "v${version}";
+    sha256 = "sha256-VNBz6roula/YlsYMM1kjQT3TLuQ86/MESzNNkEf/92M=";
+  };
+  vendorHash = null;
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postPatch = ''
+    printf ${version} > $sourceRoot/gosmee/templates/version
```

Comment:
```suggestion
    mkdir -p $sourceRoot/gosmee/templates
    printf ${version} > $sourceRoot/gosmee/templates/version
```

```
source root is source
Running phase: patchPhase
/nix/store/wq046wv3s3h4m2n9ak3j534di2pzra93-stdenv-linux/setup: line 114: source/gosmee/templates/version: No such file or directory
/nix/store/wq046wv3s3h4m2n9ak3j534di2pzra93-stdenv-linux/setup: line 131: pop_var_context: head of shell_variables not a function context
error: builder for '/nix/store/l759i5krlyliwbk7lr0jdg3cj4qn0rnf-gosmee-0.21.0.drv' failed with exit code 1;
```

Reply:
in `postPatch`, `PWD` already points to `/build/source` and `sourceRoot` points to `source`, so it fails anyway.